src/main/java/com/example/employee/repository/EmployeeRepository.java

```diff
@@ -4,4 +4,5 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface EmployeeRepository extends JpaRepository<Employee, Long> {
+    Employee findEmployeeByFirstNameAndLastNameAndDepartmentAndPositionsAndEmailAndIsActiveAndIsPensionsPayer(String first_name, String last_name, String department, String position, String email, Boolean is_active, Boolean is_pensions_payer);
```

Comment:
If we will change the way we check for existing employees we will also need to adjust this method